TA-microsoft-graph-security-score/appserver/static/js/build/globalConfig.json

```diff
@@ -5,7 +5,7 @@
         "version": "1.0.1",
         "apiVersion": "3.0.0",
```

Comment:
Update version to the UCC version that we used

Reply:
Updated with version 5.28.1